src/scss/components/landing/background/backdrop/hero/_hero.scss

```diff
@@ -37,11 +37,11 @@
     position: absolute;
     // width: 1002px;
```

Comment:
Let's try to not have commented out code. If it's not in use let's not have it in the source code.

Reply:
yeah, i'll start removing those comments as we move forward. some of them were reminders of what used to work.